library/cartridge_get_disabled_instances.py

```diff
@@ -0,0 +1,78 @@
+#!/usr/bin/env python
+
+from ansible.module_utils.helpers import Helpers as helpers
+
+argument_spec = {
+    'module_hostvars': {'required': True, 'type': 'dict'},
+    'play_hosts': {'required': True, 'type': 'list'},
+}
+
+
+def get_disabled_instances(instance_vars):
```

Comment:
This function looks like it performs the main module action. I can't imagine a better name, it's smth like "instances disabled for one specific instance"?